utils/helper.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
 function check_env() {
+  if [[ $(type curl 2>&1) =~ "not found" ]]; then
```

Comment:
1. Let's pull the curl check out into its own function `check_curl()`.
2. Since the curl is called from `confluent version`, then `check_curl()` should be called from `check_running_cp()`

Reply:
Done.

There's other functions in that file that also run curl, so we could also use this function there - eg. `check_running_elasticsearch()` and `check_running_grafana()`.  Not sure if they ever get called without having first called `check_running_cp()` though.